report_generator_3.py

```diff
@@ -0,0 +1,18 @@
+import subprocess
+
+def generate_report(report_title, content_file):
+    """Generates a PDF report with a user-supplied title."""
+    
+
+    command = f"pandoc {content_file} -o report.pdf --metadata title='{report_title}'"
```

Comment:
HIGH Command Injection: User-controlled input `report_title` is directly embedded into a shell command without proper sanitization or escaping, leading to a command injection vulnerability. An attacker can execute arbitrary commands. Ensure `import shlex` is added at the top of the file.

```suggestion
command = f"pandoc {content_file} -o report.pdf --metadata title={shlex.quote(report_title)}"
```